tox.ini

```diff
@@ -59,20 +59,20 @@ commands = pytest -v --basetemp=_pytest_temp -s -m "gtkui" deluge/tests
 commands = pytest -v --basetemp=_pytest_temp -s -m "todo" deluge/tests
 
 [testenv:trial]
-setenv = {[testenv]setenv}:{toxinidir}/deluge/tests
+setenv = {[testenv]setenv};{toxinidir}/deluge/tests
```

Comment:
Should the separator be `{:}` to be cross platform compatible? https://tox.readthedocs.io/en/latest/config.html#globally-available-substitutions

I've barely followed this PR, so please ignore me if I'm off base.

Reply:
Im not sure in general about the cross platform of the whole tox file still have to fix the depends too. Have not really tried a whole lot so I dont know much either though @cas-- mentioned about the separator earlier. Any thoughts on it much appreciated.